libnavui-androidauto/src/main/java/com/mapbox/androidauto/internal/extensions/MapboxCarMapSurfaceExtensions.kt

```diff
@@ -1,7 +1,7 @@
 @file:JvmName("MapboxCarMapSurfaceEx")
 @file:OptIn(MapboxExperimental::class)
 
-package com.mapbox.androidauto.car.internal.extensions
+package com.mapbox.androidauto.internal.extensions
```

Comment:
`MapboxCarMapSurfaceExtensions` doesn't need to be under an `internal` package as all the extensions are already marked with the `internal` modifier.

Reply:
Yeah, these are `internal` functions because we do not have examples downstream that use it yet. It is in the `internal` package because the public api is undecided.